index.html

```diff
@@ -7,27 +7,30 @@
   <title>Rose's Portfolio - A full-stack Webdeveloper</title>
   <link rel="stylesheet" href="styles.css">
 </head>
-<body class="body-content">
+<body class="body-content" id="body">
   <header class="header">
       <div class="logo-section">
         <a href="/" class="logo">Rose</a>
       </div>
       <nav class="nav">
-        <div class="bars">
+        <div class="bars" onclick="OpenNavBarMenu()" id="hamburger">
```

Comment:
It is a [requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md) that When the user clicks (or taps) the hamburger button, the mobile menu appears, but currently, this functionality doesn't work on your app. kindly fix it👍

<img width="533" alt="mobile menu appear on click" src="https://user-images.githubusercontent.com/64643995/223185354-c89eaf37-26ab-4635-83d2-c2ee0ecafe54.png">

Kindly read through the[ requirements ](https://github.com/rosemutai/My-Portfolio/pull/9/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R16)of the project and ensure you stick to the guidelines😉😉